frontend/src/views/main/project/SuiteTestExecutionList.vue

```diff
@@ -136,5 +140,54 @@ async function removeTest(suiteTest: SuiteTestDTO) {
     }
   });
 }
+
+async function debugTest(result: SuiteTestExecutionDTO, suiteTest: SuiteTestDTO) {
+  let inputs = [];
+  if (props.execution.inputs.length > 0) {
+    inputs = props.execution.inputs;
+  } else {
+    // This is a bit hacky, but there is no other way because the test suite input list is never guaranteed to be present :(
+    inputs.push({
+      name: "model",
+      type: "BaseModel",
+      isAlias: false,
+      value: result.inputs["model"],
+      params: []
+    });
+
+    inputs.push({
+      name: "dataset",
+      type: "Dataset",
+      isAlias: false,
+      value: result.inputs["dataset"],
+      params: []
+    });
+
+    // name: arg.name,
+    // type: arg.type,
+    // isAlias: false,
+    // value: '',
+    // params: []
+  }
+
+  let model = inputs.filter(i => i.name == "model")[0].value;
+
+  let res = await api.runAdHocTest(testSuiteStore.projectId!, suiteTest.testUuid, inputs, true);
+  let dataset = res.result[0].result.outputDfUuid;
+
+  const debuggingSession = await api.prepareInspection({
```

Comment:
you should also pass a project key. It'll become unreliable to take project id from either dataset or model as soon as we decide to share models/datasets across projects. Where possible we try to specify a project key explicitly

Reply:
not doing it ATM